src/openassetio-core-c/private/handles/managerAPI/ManagerInterface.hpp

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2013-2022 The Foundry Visionmongers Ltd
+#pragma once
+
+#include <openassetio/c/managerAPI/ManagerInterface.h>
+#include <openassetio/export.h>
+
+#include <openassetio/managerAPI/ManagerInterface.hpp>
+
+#include "../Converter.hpp"
+
+namespace openassetio {
+inline namespace OPENASSETIO_VERSION {
+namespace handles::managerAPI {
+using ManagerInterface = Converter<openassetio::managerAPI::ManagerInterfacePtr,
```

Comment:
[question] Should we use a naming convention that differentiates shared ownership handles from the others?

Reply:
Hmm, how about we make it more consistent to become `using ManagerInterfacePtr = ...`? 

The only time we would explicitly suffix with `Ptr` would be the `SharedPtr` case.  Though at-a-glance a new user wouldn't necessarily make that connection.

`using ManagerInterfaceSharedPtr = ...`, then?

Reply:
I think you're right about the suffix ambiguity for the "normal" case. How about
`using SharedManagerInterface = ...` as we're already in the `handles` space?

Reply:
Updated to your suggestion, both converter and handle, in 85e62d85.  I'm not sure about this, but then we need to explore more what the lifecycle of a `managerAPI_SharedManagerInterface_h` looks like, and if it even needs to exist.